src/main/java/com/impactupgrade/integration/hubspot/ContactsClient.java

```diff
@@ -44,4 +54,72 @@ public Contact insert(ContactBuilder contactBuilder) throws DuplicateContactExce
       return response.readEntity(Contact.class);
     }
   }
+
+  public Contact update(
+    String email,
+    String first_name,
+    String last_name,
+    String home_phone,
+    String mobile_phone,
+    String work_phone,
+    String preferred_phone,
+    String address,
+    String city,
+    String state,
+    String zip,
+    String country) {
+
+    ContactBuilder cBuilder = new ContactBuilder();
+    cBuilder.firstName(first_name);
+    LOGGER.log(Level.INFO, first_name);
+    cBuilder.lastName(last_name);
+    LOGGER.log(Level.INFO, last_name);
+
+    cBuilder.home_phone(home_phone);
+    LOGGER.log(Level.INFO, home_phone);
+    cBuilder.mobile_phone(mobile_phone);
+    LOGGER.log(Level.INFO, mobile_phone);
+    cBuilder.work_phone(work_phone);
+    LOGGER.log(Level.INFO, work_phone);
+    cBuilder.preferred_phone(preferred_phone);
+    LOGGER.log(Level.INFO, preferred_phone);
+
+    if (preferred_phone == "Home") {
```

Comment:
For all of the below checks, this format is safer:

`"Home".equalsIgnoreCase(preferredPhone)`

It's case insensitive, as well as prevents NPEs if preferredPhone is null.